docker-compose-tor.yml

```diff
@@ -0,0 +1,67 @@
+version: "3.7"
+
+services:
+  # TDEX daemon connected to Blockstream.info explorer
+  tdexd:
+    container_name: "tdexd"
+    image: ghcr.io/tdex-network/tdexd:latest
+    restart: unless-stopped
+    environment:
+      - TDEX_LOG_LEVEL=4 # 5 for Debug. 4 for production
+      - TDEX_CRAWL_INTERVAL=5000 # 5 seconds to poll blockchain
+      - TDEX_CRAWL_LIMIT=5000
+      - TDEX_CRAWL_TOKEN=500
+      - TDEX_EXPLORER_ENDPOINT=${EXPLORER} # 5 seconds to poll blockchain
+      
+      #### Macaroons auth/TLS termination - Operator interface ####
+      # Uncomment to disable macaroons auth
+      #- TDEX_NO_MACAROONS=true
+
+      # Uncomment one or both if running on a remote host with auth enabled
+      #- TDEX_OPERATOR_EXTRA_IP=0.0.0.0
```

Comment:
This must be passed with the machine IP so it can generate TLS certtificates for authentication.